pkg/pillar/rstats/rstats_n.go

```diff
@@ -0,0 +1,9 @@
+//go:build !rstats
+// +build !rstats
+
+package rstats
+
+var Endpoint = "localhost:8125"
```

Comment:
Can you add a new document in the docs directory (or add to some existing document) an example of how to use these stats?

Will you be running a daemon listening on port 8125? If so you probably want to add a fixed iptables rule blocking external connections to that port like we do for some other ports in pkg/pillar/dpcreconciler/linux.go

Reply:
> Will you be running a daemon listening on port 8125? If so you probably want to add a fixed iptables rule blocking external connections to that port like we do for some other ports in pkg/pillar/dpcreconciler/linux.go

No not listening on it, I send data over UDP to a statsd daemon that should be running outside of eve-os, the "localhost:8125" is just a place holder that gets replaced during build by a IP. I will add a comments in the code to prevent future confusion.

Reply:
OK